tests/functional/desy/fixtures/desy_ftp_records_expected.json

```diff
@@ -1,2168 +1,466 @@
-      "format": ".txt",
-      "path": "/tmp/file_urls/full/1d4b0a4eebdd03b95f882fa7feb9d3f06681ec50.txt",
-      "type": "Main",
-      "filename": "test_fft_2"
-    }
-  ],
-  "control_number": 444444,
-  "public_notes": [
-    {
-      "value": "*Brief entry*"
-    }
-  ],
-  "self": {
-    "$ref": "https://labs.inspirehep.net/api/literature/444444"
-  },
-  "number_of_pages": 6,
-  "titles": [
-    {
-      "source": "JACoW",
-      "title": "Towards a Fully Integrated Accelerator on a Chip: Dielectric Laser\n                Acceleration (DLA) From the Source to Relativistic Electrons\n            "
-    }
-  ],
-  "dois": [
-    {
-      "value": "10.18429/JACoW-IPAC2017-WEYB1"
-    }
-  ],
-  "publication_info": [
-    {
-      "parent_isbn": "9783954501823"
-    },
-    {
-      "page_start": "2520",
-      "page_end": "2525",
-      "year": 2017
-    }
-  ],
-  "$schema": "hep.json",
-  "document_type": [
-    "article"
-  ],
-  "abstracts": [
-    {
-      "source": "Deutsches Elektronen-Synchrotron",
-      "value": "Dielectric laser acceleration of electrons has recently been\n                demonstrated with significantly higher accelerating gradients than other\n                structure-based linear accelerators. Towards the development of an integrated 1 MeV\n                electron accelerator based on dielectric laser accelerator technologies,\n                development in several relevant technologies is needed. In this work, recent\n                developments on electron sources, bunching, accelerating, focussing, deflecting and\n                laser coupling structures are reported. With an eye to the near future, components\n                required for a 1 MeV kinetic energy tabletop accelerator producing sub-femtosecond\n                electron bunches are outlined.\n            "
-    }
-  ]
-},
-{
-    "acquisition_source": {
-        "source": "desy",
-        "method": "hepcrawl",
-        "submission_number": "5652c7f6190f11e79e8000224dabeaad",
-        "datetime": "2017-05-04T17:49:07.975168"
-    },
-    "_collections": [
-        "Literature"
-    ],
-    "control_number": 1608652,
-    "public_notes": [
-        {
-            "value": "*Brief entry*"
-        }
-    ],
-    "self": {
-        "$ref": "https://labs.inspirehep.net/api/literature/1608652"
-    },
-    "number_of_pages": 6,
-    "inspire_categories": [
-        {
-            "source": "undefined",
-            "term": "Accelerators"
+[
+    {
+        "core": true, 
+        "documents": [
+            {
+                "url": "http://bib-pubdb1.desy.de/record/389977/files/desy-thesis-17-035.title.pdf", 
```

Comment:
Is this how they will pass us the ffts? Will it always be an external url? (I was actually expecting to have a relative path in the ftp server...).

Reply:
Will this url be open to the world or will it need some authentication?

Reply:
I have no idea how they will provide files. Isn't that something you discussed with them? Here the files are publicly available theses that were provided through HTTP, but you probably want to modify the test to take files from FTP.

Reply:
@fschwenn @ksachs Hey! can you clarify for me how will the fft files be passed to be parsed by hepcrawl? There's really no limitation, but I want to make sure that we properly support and test whatever methods you would like (passing an external url, a relative path inside the ftp, ...)

Thanks!

Reply:
I will go on with this pr using the url for now, will adapt it later.

Reply:
DESY internal stuff (PubDB) should always be public accessible URLs.

If we use the DESY spider for external stuff, as long as the 'real' spider for that publisher is not available, if depends on how the publisher provides the full-text. 
I guess we will have the real spider for those that push to our FTP server.
The rest is public URLs.

Reply:
@ksachs so, just to make sure, we will have:
* ftp relative paths (maybe for some of the publishers that push to DESY ftp)
* public urls  (for everybody else)

No?

Reply:
I think so.

Reply:
Perfect 👍